src/services/solana-client/index.ts

```diff
@@ -0,0 +1,22 @@
+import transferWAudioBalance from './transferWAudioBalance'
+import transferAudioIntoWormhole from './transferAudioIntoWormhole'
+import createUserBank, { getBankAccountAddress } from './createUserBank'
+import relay from './relay'
+
+export const IDENTITY_SERVICE = process.env.REACT_APP_IDENTITY_SERVICE
+
+export class SolanaClient {
```

Comment:
wondering why we have all these methods static? i'm not 100% that this will be a stateless class - I actually think it probably won't be, looking at the solana Brick demo as an example